src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/NewAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -179,8 +179,6 @@ public class NewAzureRmRecoveryServicesAsrReplicationProtectedItem : SiteRecover
         [Parameter(ParameterSetName = ASRParameterSets.HyperVSiteToAzure)]
         [Parameter(ParameterSetName = VMwareToAzureWithDiskType)]
         [Parameter(ParameterSetName = VMwareToAzureParameterSet)]
-        [Parameter(ParameterSetName = ASRParameterSets.AzureToAzure)]
```

Comment:
Removing parameters (from parameter sets) is a breaking change and is not allowed outside of major releases. The next major release starts from Sep 6th, so please
1. Defer this PR until major release
2. Submit another PR to add [breaking change attributes ](https://github.com/Azure/azure-powershell/blob/main/documentation/breaking-changes/breaking-changes-attribute-help.md)that serve as preannouncement.

Reply:
@isra-fel I am wondering if have you had a chance to check out my recent changes. Thank you so much.